lib/mini_autobot/settings.rb

```diff
@@ -93,6 +93,14 @@ def verbosity_level
       hsh.fetch(:verbosity_level, 0).to_i
     end
 
+    def google_sheets?
+      true unless hsh[:google_sheets] == false
+    end
+
+    def google_sheet
+      hsh.fetch(:google_sheets)
```

Comment:
Keep in mind that `Hash#fetch` raises a `KeyError`, when you don't provide it a second argument and the key doesn't exist. If you don't need a default value, use `Hash#[]` instead, e.g.:

```ruby
def google_sheet
  hsh[:google_sheets]
end
```